vite.config.js

```diff
@@ -25,88 +17,50 @@ export const viteConfig = (configEnv = {}) => {
     }
   }
   /**
-   * @type {typeof import('vite').UserConfigExport}
+   * @type {import('vite').UserConfigExport}
    */
   return {
     build: {
       copyPublicDir: true,
       target: 'esnext',
       outDir: 'build',
       commonjsOptions: {
-        include: [
-          /ipld-explorer-components/,
-        ],
         exclude: []
       },
       rollupOptions: {
-        treeshake: false,
-        plugins: [
-          nodeResolve({
-            browser: true,
-          }),
-          // replace({
-          //   'process.env.NODE_ENV': JSON.stringify(mode)
-          // }),
-          babel({
-            presets: [
-              "@babel/preset-react",
-              ["@babel/preset-env", {
-                "useBuiltIns": "entry",
-                "corejs": "3.22"
-              }],
-            ],
-          }),
-          commonjs({
-            include: [
-              'node_modules/**',
-            ],
-            exclude: [
-              'node_modules/process-es6/**',
-            ],
-            namedExports: {
-              'node_modules/react/index.js': ['Children', 'Component', 'PropTypes', 'createElement'],
-              'node_modules/react-dom/index.js': ['render'],
-            },
-          }),
-        ],
-      }
+        treeshake: true,
+        plugins: []
+      },
+      emptyOutDir: true
     },
     define,
     // https://vitejs.dev/guide/dep-pre-bundling.html#monorepos-and-linked-dependencies
     optimizeDeps: {
-      include: [
-        'ipld-explorer-components',
-      ],
-      exclude: [],
+      exclude: []
     },
     plugins: [
-      react(),
-      reactVirtualized(),
+      react()
     ],
     resolve: {
-      alias: [
-        { find: /^assert$/, replacement: 'assert' },
-        { find: /^os$/, replacement: 'rollup-plugin-node-polyfills/polyfills/os' },
-        { find: /^process$/, replacement: 'rollup-plugin-node-polyfills/polyfills/process-es6' },
-        { find: /^stream$/, replacement: 'rollup-plugin-node-polyfills/polyfills/stream' },
-        { find: /^util$/, replacement: 'rollup-plugin-node-polyfills/polyfills/util' },
-      ]
+      alias: []
     },
     // see https://www.robinwieruch.de/vitest-react-testing-library/
     test: {
       globals: true,
       environment: 'jsdom',
       setupFiles: './test/unit/setup.js',
       include: [
-        'src\/**\/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'
+        'src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'
       ],
-      deps: {
-        inline: [
-          "ipld-explorer-components"
-        ]
+      server: {
+        deps: {
+          inline: [
+            'ipld-explorer-components'
+          ]
+        }
       }
-    },
-  };
+    }
+  }
 }
 
-export default defineConfig(viteConfig);
+export default defineConfig(viteConfig)
```

Comment:
we no longer needed a lot of this since the ipld-explorer-components is updated and outputting a more usable bundle now